src/arrow-sdk.ts

```diff
@@ -261,65 +261,87 @@ export async function getStrikeGrid(
 }
 
 /**
- * Submit an option order to the API to compute the live price and submit a transaction to the blockchain.
+ * Submit multiple option orders to the API to compute the live price and submit a transaction to the blockchain.
+ *
+ * @param deliverOptionParams[] Object containing parameters necessary to create an option order on Arrow.
```

Comment:
Can you update this to be `@param deliverOptionParams[] Array of objects containing parameters ...`?